tests/Base.php

```diff
@@ -209,10 +209,20 @@ public function testRequestMethod(): void
         self::assertEquals(200, $response['code']);
         // Bug in C++ framework makes this an empty string
         // self::assertEquals('OPTIONS', $response['body']);
+        self::assertContains($response['body'], ['OPTIONS', '']);
 
         $response = $this->execute(method: 'PATCH', headers: ['x-action' => 'requestMethod']);
         self::assertEquals(200, $response['code']);
         self::assertEquals('PATCH', $response['body']);
+
+        $response = $this->execute(method: 'HEAD', headers: ['x-action' => 'requestMethod']);
+        self::assertEquals(200, $response['code']);
+        //  var_dump says the body is a boolean in C++.
+        self::assertContains($response['body'], ['HEAD', '', false]);
```

Comment:
@Meldiron,
One thing to note here is that if I do below, the body is properly an empty string -
```php
self::assertContains(trim($response['body']), ['HEAD', '']);
```
else its a boolean, atleast as per the `var_dump`.

Reply:
Something similar happens on the Kotlin side as well -
```php
["body"]=> bool(false)
```